index.html

```diff
@@ -137,9 +135,7 @@ <h3 class="tertiary-heading">Tonic</h3>
       </div>
```

Comment:
- [ ] According to the project requirements, the projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript. Check an example of students' projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636.

Reply:
@mikethreels Done in Removing the projects list in index.html and updated Javascript.